package.json

```diff
@@ -219,6 +219,8 @@
     "**/**/socket.io": "^2.4.0",
     "**/**/y18n": "^3.2.2",
     "**/**/ssri": "8.0.1",
-    "**/**/jose": "1.28.1"
+    "**/**/jose": "1.28.1",
+    "**/**/**/ssri": "^8.0.1",
```

Comment:
@leepavlouhmcts - is this correct, i.e. `**/**/**/` (3 of them)?

Reply:
replaced **/**./**/** to **/**/